pilot/pkg/security/model/authentication_test.go

```diff
@@ -23,14 +23,15 @@ import (
 	"github.com/envoyproxy/go-control-plane/envoy/config/grpc_credential/v2alpha"
 	"github.com/gogo/protobuf/types"
 
+	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pkg/features/pilot"
 )
 
 func TestParseJwksURI(t *testing.T) {
```

Comment:
Could you also move the unit test to `jwks_resolver_test.go` if `TestParseJwksURI` is moved to `jwks_resolver.go`

Reply:
Is this still relevant if the jwks packages are not moving?

Reply:
Yes, I mean you put the `ParseJwksURI` function in `jwks_resolver.go`, so its unit test (`TestParseJwksURI`) should also be in `jwks_resolver_test.go`, instead of in a different file `authentication_test.go`.

Reply:
This test function uses type pilot/pkg/security/model::Port, which would cause the pilot/pkg/model package to depend on pilot/pkg/security/model, which is the dependency path this PR attempts to break.  Then again, the dependency is only in tests, so maybe that wouldn't impact Galley?  I will attempt this change and wait to see if Galley re-imports all of pilot as a result...

Reply:
I attempted this change, but go will not even compile it, as it creates a circular dependency between these packages.  I think that test will have to stay where it is or be refactored to not use the model.Port type...

Reply:
I think that can be refactored later.